packages/analytics/package.json

```diff
@@ -35,7 +35,8 @@
     "ember-uikit": "^9.1.1",
     "ember-validated-form": "^7.0.1",
     "graphql": "^15.8.0",
-    "reactiveweb": "^1.2.2"
+    "reactiveweb": "^1.2.2",
+    "xlsx": "https://cdn.sheetjs.com/xlsx-0.20.2/xlsx-0.20.2.tgz"
```

Comment:
Pretty odd package, do you have any gripes with this @anehx ? 
Would it make sense to have this in the backend?

Reply:
Yes, it would make sense to do this in the backend. The problem is, our current analytics approach does not scale. Implementing a new endpoint in core caluma for something we know we'll have to get rid of just doesn't make any sense.
The approach here is pretty much copy/paste from something @StephanH90 did in another project.